jcl/j9modules.xml

```diff
@@ -83,6 +83,14 @@
 					</else>
 				</if>
 			</case>
+			<case value="com.ibm.dataaccess">
+				<removeexport package="com.ibm.dataaccess" />
+				<delete verbose="true">
+					<fileset dir="${module.dir}/classes">
+						<include name="com/ibm/dataaccess/*" />
+					</fileset>
+				</delete>
+			</case>
```

Comment:
Is this script required? package `com.ibm.dataaccess` export is removed within module `java.base` in same xml file, and there is no change to the actual package/class files.

Reply:
This script doesn't seem to be needed. Will remove.